theano/gpuarray/opt.py

```diff
@@ -264,8 +267,18 @@ def add_requirements(self, fgraph):
         fgraph.attach_feature(toolbox.ReplaceValidate())
 
     def apply(self, fgraph):
+        change_tracker = ChangeTracker()
         mapping = {}
-
+        global_process_count = {}
```

Comment:
yeah, but I am using it to count the local optimizers. EquilibruimOptimizer also did this at line 2256. That is why I have this.